db/people.yml

```diff
@@ -77,6 +79,14 @@ cwright:
       gpa: 3.49
   email: cjwright4242@gmail.com
   employment:
+    - begin_year: 2016
+      location: New York, NY
+      organization: columbiau
+      position: Graduate Student, Lead Software Engineer
+      other:
+        - 'Lead Software Engineer specializing in streaming data analysis, data provanance and software packaging'
+        - 'Lead a team of software developers, creating solutions for the data acquesition and analysis needs
```

Comment:
this is not your job description....I would prefer it if you added this below your main roles.  This would be your job if you were working for NSLS-II core budget....

Reply:
What do you mean "below your main roles"?